packages/storage-plus/src/indexed_map.rs

```diff
@@ -722,31 +722,31 @@ mod test {
             last_name: "".to_string(),
             age: 42,
         };
-        let pk1: &[u8] = b"5627";
+        let pk1: &str = "5627";
```

Comment:
I think those type annotations are unnecessary here, `&str` is default type for string slice.